databases/ClickHouse-server/Portfile

```
@@ -0,0 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                ClickHouse-server
+version             1
+
+description         Run ClickHouse® as a system service.
+long_description    ${description}
+
+categories          databases
+maintainers         @slarew openmaintainer
```

We recommend listing your obfuscated email as well as your GitHub handle in the `maintainers` line.
```suggestion
maintainers         {slarew.net:stephen @slarew} openmaintainer
```